manifests/init.pp

```diff
@@ -226,7 +226,7 @@
   String[1] $db_password = $foreman::params::db_password,
   Optional[String[1]] $db_sslmode = undef,
   Optional[String[1]] $db_root_cert = undef,
-  Integer[0] $db_pool = 5,
+  Optional[Integer[0]] $db_pool = undef,
```

Comment:
On line 87 there's documentation for this parameter that's shown in the installer when using `--full-help`. Perhaps you can update it to describe the behavior.